crawler/store/kaufland.py

```diff
@@ -37,8 +37,7 @@ class KauflandCrawler(BaseCrawler):
         "quantity": ("neto količina(KG)", False),
         "unit": ("jedinica mjere", False),
         "barcode": ("barkod", False),
-        "category": ("Kategorija", False),
-        "anchor_date": ("Datum sidrenja", False),
+        "category": ("WG", False),
```

Comment:
CW (from commit message) or WG (in the code)?

Reply:
I doublechecked and it's WG :)